prep-css-display/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<head>
+  <title>CSS Display Properties</title>
+  <style>
+</head>
+      <div> I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
```

Comment:
The div and ul elements should all go in the body of the document, you can see an example here

https://github.com/gabriellerramos/prep-0922-code-solutions/pull/8#pullrequestreview-1122856949